src/Microsoft.Identity.Web/MicrosoftGraph/TokenAcquisitionCredentialProvider.cs

```diff
@@ -1,14 +1,16 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Microsoft.Graph;
-using Microsoft.Identity.Web;
 using System.Collections.Generic;
 using System.Net.Http;
 using System.Threading.Tasks;
+using Microsoft.Graph;
 
-namespace blazor
+namespace Microsoft.Identity.Web
 {
+    /// <summary>
+    /// Authentication provider based on MSAL.NET.
+    /// </summary>
```

Comment:
nit: this comment seems a little misleading.

Reply:
Addressed